benchmarks/src/main/scala/zio/stm/TReentrantLockBenchmark.scala

```diff
@@ -0,0 +1,104 @@
+package zio.stm
+
+import java.util.concurrent.TimeUnit
+import java.util.concurrent.locks.StampedLock
+
+import scala.util.Random
+
+import org.openjdk.jmh.annotations._
+
+import zio.IOBenchmarks._
+import zio.{ Managed, _ }
+@State(Scope.Benchmark)
+@BenchmarkMode(Array(Mode.Throughput))
+@OutputTimeUnit(TimeUnit.SECONDS)
+@Measurement(iterations = 15, timeUnit = TimeUnit.SECONDS, time = 5)
+@Warmup(iterations = 15, timeUnit = TimeUnit.SECONDS, time = 5)
+@Fork(1)
+@Threads(1)
+class TReentrantLockBenchmark {
+
+  @Param(Array("50", "100", "500"))
+  var numReaders: Int = _
+
+  @Param(Array("1", "10", "25", "50", "100"))
+  var numWriters: Int = _
+
+  @Param(Array("100"))
+  var ops: Int = _
```

Comment:
It can be constant.

Reply:
@mijicd - thanks for your feedback. I will fix and create new PR soon. Qs - Can I see the results of all the benchmarks in CircleCI? Do you have previous results saved too somewhere?

Reply:
No need for a new one, just fix here :). Regarding benchmark results, they're not visible on circleci, as they're not running there to the best of my knowledge. However, we should find some usable plan to run them more frequently, as I believe there will be "surprises".